code/selection_algorithms/src/quick_select.kt

```diff
@@ -0,0 +1,43 @@
+const val MAX = Int.MAX_VALUE
```

Comment:
You should put more empty lines in your code.

Reply:
yeah, i make sure that I will work in accordance with the information provided by guide on style of writing codes.

Reply:
@abdouskamel I have updated this file have a loook.